packages/core/src/http-client/http-client.ts

```diff
@@ -213,3 +223,53 @@ export function getAxiosConfigWithDefaultsWithoutMethod(): Omit<
     httpsAgent: new https.Agent()
   };
 }
+
+function getDefaultHttpRequestOptions(): HttpRequestOptions {
+  // TODO: 2.0 change to true
```

Comment:
Can you add this information to the api doc as well?

Reply:
I guess you mean the js doc?